doc/lua_api.md

```diff
@@ -486,8 +486,7 @@ stripping out the file extension:
 * e.g. `foomod_foothing.png`
 * e.g. `foomod_foothing`
 
-Supported texture formats are PNG (`.png`), JPEG (`.jpg`), Bitmap (`.bmp`)
-and Targa (`.tga`).
+Supported texture formats are PNG (`.png`), JPEG (`.jpg`) and Targa (`.tga`).
 Since better alternatives exist, the latter two may be removed in the future.
```

Comment:
```suggestion
Since better alternatives exist, the latter may be removed in the future.
```

we don't want to deprecate JPEG, do we?

Reply:
better to remove this I think. there's no current consensus to deprecate tga

Reply:
Sure.